models/optimism_attestationstation/optimism/optimism_attestationstation_optimism_events.sql

```diff
@@ -26,39 +27,40 @@ SELECT
         ,creator as issuer
         ,contract_address
         ,key as key_raw
-        ,
-        REGEXP_REPLACE(--Replace invisible characters
-            decode(
-                unhex(
-                  if (
-                    substring(key, 1, 6) in ("0xab7e", "0x9e43"), --Handle for Clique
-                    hex(key),
-                    substring(key, 3)
-                  )
-                ),
-                "utf8"
-              ) 
-            , '[^\x20-\x7E]','')
-        as key
-
+        ,regexp_replace(
+          -- Replace invisible characters
+          from_utf8(
+              from_hex(
+                  CASE 
+                      WHEN bytearray_substring(key, 1, 2) IN (0xab7e, 0x9e43) -- Handle for Clique
+                      THEN to_hex(key)
+                      ELSE substring(cast(key as varchar), 3)
+                  END
+              )
+          ),
+          '[^\x20-\x7E]',
+          ''
+        ) as key
         ,val as val_raw
 
         ,split(
                 REGEXP_REPLACE(--Replace invisible characters
-                        CASE WHEN cast( REGEXP_REPLACE(unhex(substring(val, 3)), '[^\x20-\x7E]','') as varchar(100)) != ""
-                            THEN cast(unhex(substring(val, 3)) as varchar(100))
-                            ELSE cast(bytea2numeric_v3(substring(val, 3)) as varchar(100))
+                        CASE WHEN cast( REGEXP_REPLACE(from_utf8(val), '[^\x20-\x7E]','') as varchar(100)) != ''
+                            THEN cast(from_utf8(val) as varchar(100))
+                            ELSE cast(bytearray_to_uint256(if(bytearray_length(val) > 32, bytearray_substring(val, 1, 32), val)) as varchar(100))
```

Comment:
i pushed a commit here, to fix missing `)` and cast outputs, so now we should be good